filament/src/ResourceAllocator.h

```diff
@@ -90,7 +90,7 @@ class ResourceAllocator final : public ResourceAllocatorInterface {
 
 private:
     // TODO: these should be settings of the engine
-    static constexpr size_t CACHE_CAPACITY = 64u << 20u;   // 64 MiB
+    static constexpr size_t CACHE_CAPACITY = 640u << 20u;   // 640 MiB
```

Comment:
@veghbalazs  I have two questions regarding this:
 - Is this needed on iOS too?
 - Is 640 MB a proper value for production code? Isn't it too high?

Reply:
We discussed this with Balazs on Slack. I played around with this value and found the following:
 - This cache doesn't contain any of the meshes' textures only the render targets. 
 - On my iPad (which has a resolution of 2732x2048 px), this cache contains:
   -  1024x1024 _Shadow Map_ with DEPTH16 format
   -  2736x2048 _Color Buffer_ with RGBA16 format
   -  2736x2048 _Depth Buffer_ with DEPTH32 format
   -  2732x2048 _Tonemapped Buffer_ with RGBA8 format
   which adds up to ~87.5 MB. So 640 MB is too much, 64 MB was probably optimized for phones (which have lower resolution). I chose 128 MB for now (see 2fb6c9bf1bf2e99d1c490f8131d64794b0f492dc).
 - Increasing this capacity constant doesn't seem to increase the memory consumption of the app, so probably nothing is preallocated.

Reply:
128 MB will be enough for all of the iPad models, because the one I tested with (iPad Pro 3rd Gen) [is already one of the biggest](https://www.ios-resolution.com/).